### ISSUE TITLE
Will Bluesky social network, even if it has its own protocol, be compatible with the ActivityPub protocol?

### ISSUE THREAD

**Issue by goliv04053** [bluesky-social/atproto#345]
Someone made a question on #255 (Already a decentralized federated protocol) talking: "_Activitypub already exists. Why not just work on that? Why is this needed?_". The user and collaborator @pfrazee already answered the motifs of why it's being developed a new protocol. But now, the question that I have is: Even with the implementation of the AT (Authenticated Transfer Protocol) in Bluesky, will Bluesky be compatible with the ActivityPub protocol? 

TL;DR: Will both protocols be available on BlueSky?

**Comment by deanpress** [comment]
This is a repository for AT Protocol, not the Bluesky client.

**Comment by goliv04053** [comment]
Sorry for bothering, but where can I ask this question?

**Comment by pfrazee** [comment]
I'll just answer here: currently, no there's no plan for AP integration in the Bluesky client.

**Comment by eloquence** [comment]
@pfrazee Any plans to revisit this question? It would suck to end up with two federated platforms that have traction but can't talk to each other.

**Comment by eloquence** [comment]
Since this issue is old & closed, I've left a longer note here:
https://github.com/bluesky-social/atproto/discussions/1716
